dir.props

```diff
@@ -225,7 +225,7 @@
     <NetFxPackageRuntimePath>$(BinDir)pkg\netfx\lib</NetFxPackageRuntimePath>
 
     <!-- We add extra binplacing for the test shared framework until we can get hardlinking with the runtime directory working on all platforms -->
-    <BinPlaceTestSharedFramework Condition="'$(_bc_TargetGroup)' == 'netcoreapp'">true</BinPlaceTestSharedFramework>
+    <BinPlaceTestSharedFramework Condition="'$(_bc_TargetGroup)' == 'netcoreapp' AND '$(BuildTests)'=='true'">true</BinPlaceTestSharedFramework>
```

Comment:
Can you please make the condition `$(BuildTests)' != 'false`? See comments in https://github.com/dotnet/corefx/pull/24034 for reasons.